src/test/java/test/thread/parallelization/BaseParallelizationTest.java

```diff
@@ -204,8 +204,8 @@ public static void verifyEventThreadsSpawnedAfter(Long earlierThreadId, List<Eve
     public static void verifyEventsOccurBetween(EventLog earlierEventLog, List<EventLog> inBetweenEventLogs, EventLog
             laterEventLog, String failMessage) {
         for (EventLog eventLog : inBetweenEventLogs) {
-            assertTrue(eventLog.getTimeOfEvent() > earlierEventLog.getTimeOfEvent() &&
-                    eventLog.getTimeOfEvent() < laterEventLog.getTimeOfEvent(), failMessage);
+            assertTrue(eventLog.getTimeOfEvent() >= earlierEventLog.getTimeOfEvent() &&
```

Comment:
The tests that I added had methods with different priorities that I wanted to look at starting at the same time. I didn't think changing this from `>` to `>=` was a big change, but if you think it is, we can discuss. I can modify my test slightly if need be.